docs/installation/viam-server-setup.md

```diff
@@ -0,0 +1,125 @@
+title: "viam-server Installation Guide"
+linkTitle: "viam-server setup"
+childTitleEndOverwrite: "Try Viam"
+weight: 9
+no_list: true
+type: docs
+images: ["/installation/thumbnails/install.png"]
+imageAlt: "Install Viam"
+description: "To use Viam, install and run viam-server or viam-micro-server on the computer or microcontroller that will run your machine and is connected to your hardware."
+date: "2024-08-19"
+# updated: ""  # When the content was last entirely checked
+aliases:
+  - /installation/prepare/
+  - /installation/macos-install/
+  - /installation/linux-install/
+  - /installation/install/
+  - /installation/install/linux-install/
+  - /installation/install/macos-install
+  - /getting-started/installation/
+  - /getting-started/macos-install/
+  - /getting-started/linux-install/
+  - /installation/
+  - /get-started/installation/
+---
+
+To use Viam, you need to install either the `viam-server` binary or `viam-micro-server`.
+
+[`viam-server`](/architecture/rdk/) is the binary built from the [Robot Development Kit](https://github.com/viamrobotics/rdk) that contains and manages communications between all Viam's built-in hardware drivers ({{< glossary_tooltip term_id="component" text="components" >}}) and software {{< glossary_tooltip term_id="service" text="services" >}}, connects your machine to the cloud, manages machine configuration, and manages dependencies including {{< glossary_tooltip term_id="module" text="modules" >}}.
+
+`viam-micro-server` is a lightweight version of `viam-server` which can run on resource-limited embedded systems that cannot run the fully-featured `viam-server`.
+To install `viam-micro-server`, see [`viam-micro-server` Installation Guide](/installation/viam-micro-server-setup/).
+
+For an overview of the Viam software platform, see [Learn about Viam](/get-started/).
+
+{{< alert title="In this page" color="note" >}}
+{{% toc %}}
+{{< /alert >}}
+
+## Platform requirements
+
+`viam-server` can run on any computer that runs a supported operating system:
+
+- Linux 64-bit operating systems running on the `aarch64` or `x86_64` architectures
+- Windows Subsystem for Linux (WSL)
+- macOS
+
+{{% alert title="Windows Support Notice" color="note" %}}
+
+WSL itself does not currently support exposing many types of Windows hardware to the embedded Linux kernel.
+This means that some hardware, such as a connected webcam, may not be available to `viam-server` with WSL, even though it is fully supported for native Linux systems.
+
+Although you cannot access all hardware if you run `viam-server` on your personal computer with WSL, you can [run code to control a machine](/sdks/#run-code) on your personal computer with WSL if you install `viam-server` on a single-board computer running Linux.
+
+{{% /alert %}}
+
+### Operating System installation
+
+`viam-server` can run on **any computer that runs a [supported operating system](#platform-requirements)**, including single-board computers (SBCs) running 64-bit Linux.
```

Comment:
I think the link back is otherwise confusing since the heading doesn't match
```suggestion
`viam-server` can run on **any computer that runs on a [supported platform](#platform-requirements)**, including single-board computers (SBCs) running 64-bit Linux.
```